Core/Src/datalog_acquisition.c

```diff
@@ -16,13 +16,16 @@
 #include "util.h"
 #include "CMSIS_extra/global_variables_handler.h"
 
+uint32_t event_flag;
```

Comment:
`static`
https://www.c-programming-simple-steps.com/static-keyword-in-c.html. Da uma olhada na seção “Using the C static keyword outside of a function"